inst/qml/auditClassicalEvaluation.qml

```diff
@@ -19,456 +19,676 @@
+				enabled:						flagCriticalTransactions.checked
 
 				RadioButton 
 				{ 
-					name: "ratioBound"
-					text: qsTr("Ratio")
-					id: ratioBound
-					enabled: !variableTypeCorrect.checked 
+					text: 						qsTr("Keep")
+					name: 						"inspect"
+					checked: 					true	
 				}
 
 				RadioButton 
 				{ 
-					name: "regressionBound"
-					text: qsTr("Regression")
-					id: regressionBound
-					enabled: !variableTypeCorrect.checked
+					text: 						qsTr("Remove")
+					name: 						"remove"
 				}
 			}
+		}
+	}
 
-			RadioButtonGroup
-			{
-				title: 	qsTr("Estimation Method")
-				name: 	"estimator2"
-				visible: variableTypeCorrect.checked
-				
-				RadioButton 
-				{ 
-					name: "binomialBound"
-					text: qsTr("Binomial")
-					id: binomialBound
-					enabled: variableTypeCorrect.checked
-				}
-				
-				RadioButton 
-				{ 
-					name: "poissonBound"
-					text: qsTr("Poisson")
-					id: poissonBound
-					enabled: variableTypeCorrect.checked
-				}
-				
-				RadioButton 
-				{ 
-					name: "hyperBound"
-					text: qsTr("Hypergeometric")
-					id: hyperBound
-					enabled: variableTypeCorrect.checked 
-				}
+	Section
+	{
+		text: 									qsTr("B.     Risk Assessments")
+		columns:								3
+		enabled:								((performanceMateriality.checked & ((materialityRelative.checked & materialityPercentage.value > 0) | (materialityAbsolute.checked & materialityValue.value > 0 & populationValue.value > 0))) | (minimumPrecision.checked & minimumPrecisionPercentage.value > 0)) & populationSize.value > 0
+
+		RadioButtonGroup
+		{
+			id: 								ir
+			title: 								qsTr("Inherent Risk")
+			name: 								"IR"
+			enabled:							performanceMateriality.checked
+
+			RadioButton 
+			{ 
+				text: 							qsTr("High")
+				name: 							"High"
+				checked: 						true	
 			}
 
-			RadioButtonGroup
+			RadioButton 
+			{ 
+				text: 							qsTr("Medium")
+				name: 							"Medium"
+			}
+
+			RadioButton 
+			{ 
+				text: 							qsTr("Low")
+				name: 							"Low"
+			}
+
+			RadioButton
 			{
-				id: 		ir
-				title: 		qsTr("Inherent Risk")
-				name: 		"IR"
-
-				RadioButton { text: qsTr("High"); 		name: "High"; checked: true	}
-				RadioButton { text: qsTr("Medium");		name: "Medium"}
-				RadioButton { text: qsTr("Low"); 			name: "Low"}
-				RadioButton
-				{
-					id: 								irCustom
-					text:	 							qsTr("Custom")
-					name: 							"Custom"
-					childrenOnSameRow: 	true
+				id: 							irCustom
+				text:	 						qsTr("Custom")
+				name: 							"Custom"
+				childrenOnSameRow: 				true
 
-					PercentField
-					{
-						name: 						"irCustom"
-						visible: 					irCustom.checked
-						decimals: 				2
-						defaultValue: 		100
-						min: 							25
-					}
+				PercentField
+				{
+					name: 						"irCustom"
+					visible: 					irCustom.checked
+					decimals: 					2
+					defaultValue: 				100
+					min: 						25
 				}
 			}
+		}
 
-			RadioButtonGroup
+		RadioButtonGroup
```

Comment:
the radio button group with High/Medium/Low/Custom options is duplicated couple of times, perhaps make it a custom component?